packages/@okta/vuepress-plugin-my-okta/clientRootMixin.js

```diff
@@ -0,0 +1,59 @@
+export default {
+  mounted() {
+    let iframe = document.createElement('iframe')
+    iframe.id = 'myOktaIFrame'
+    iframe.src = 'https://login.okta.com'
+    iframe.style = 'display:none'
+    document.body.appendChild(iframe)
+
+    window.addEventListener('message', this.receiveMessage, false);
+    document.addEventListener('visibilitychange', this.checkSuccess, false);
+
+    window.addEventListener("load", () => {
+      document.querySelector('.content').innerHTML = document.querySelector('.content').innerHTML.replace(/https:\/\/{yourOktaDomain}/gi, '<span class="okta-preview-domain">https://{yourOktaDomain}</span>')
+      this.getMyOktaAccounts()
+    })
+  },
+
+  methods: {
+    checkSuccess: () => {
+      if (typeof document.hidden !== "undefined") {
+        if (!document.hidden) {
+          if ( document.querySelector('.okta-preview-domain').innerHTML == 'https://{yourOktaDomain}' ) {
+            // We haven't replaced yet: try again.
+            this.getMyOktaAccounts();
+          } else {
+            // We've succeeded: don't try again in the future.
+            document.removeEventListener('visibilitychange', this.checkSuccess, false);
+          }
+        }
+      }
+    },
+
+    getMyOktaAccounts: () => {
+      document.querySelector('#myOktaIFrame').contentWindow.postMessage({messageType: 'get_accounts_json'}, 'https://login.okta.com')
```

Comment:
nit: semicolon

Reply:
Actually, many lines in this file are missing semicolons.

Reply:
Do we need semicolons?    A lot of the sample code and vuepress code goes without... Isn't it the es6 way?

Reply:
summarizing our slack conversation for posterity (below is paraphrase):

Me: My rule has always been "If you can't describe the situation where ASI will screw you over (and there is one), then you shouldn't be relying on it".

You: This seems to be the VuePress convention, so it makes sense to follow that.

Me: (single tear silently rolls down my face)